static/src/javascripts/projects/common/modules/commercial/acquisitions-ophan.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import ophan from 'ophan/ng';
+
+const submitComponentEvent = (componentEvent: OphanComponentEvent) => {
+    ophan.record({ componentEvent });
+};
+
+// Hmmm, what about where the campaign code is different for each button?
```

Comment:
This is a more general issue, but I _think_ we agreed that each component should have at most one campaign code associated with it, in which case this is fine.